packages/build/src/config/config.ts

```diff
@@ -7,6 +8,18 @@ interface ManPageConfig {
   fileName: string;
 }
 
+// TODO: this is duplicated in update-notification-manager.ts
+export interface GreetingCTADetails {
+  chunks: {
+    text: string;
+    style?: string; // TODO: this is actually clr.ts/StyleDefinition
+  }[];
+}
+
+export type CTAConfig = {
+  [version: string | '*']: GreetingCTADetails;
```

Comment:
```suggestion
  [version: `${number}.${number}.${number}` | '*']: GreetingCTADetails;
```
(if that makes sense)

Reply:
The version can be any semver string, which includes prereleases. The permitted format is enforced by a regex in the json schema definition, so leaving it as a string is probably easiest here.